README.md

```diff
@@ -13,8 +13,10 @@ cd xsens_streaming
 mkdir build && cd build
 cmake ..
 make -j8
-sudo make install
+sudo -E make install
 ```
+Replace `cmake ..` with `cmake --DPYTHON_BINDING ..` to compile with Python bindings. 
```

Comment:
Bindings are "ON" by default, so this should be:

> Replace `cmake ..` with `cmake --DPYTHON_BINDING=OFF ..` to compile without Python bindings.

Reply:
Ok! Shall we enable bindings by default?